src/client/src/rt-util/getEnvironment.ts

```diff
@@ -0,0 +1,8 @@
+export function getEnvironment(): string | undefined {
+    const serviceUrl = process.env.REACT_APP_BROKER_HOST || window.location.host;
+
+    if (serviceUrl.includes('localhost')) return 'localhost';
+
+    const envMatch = /web-(?<env>[a-zA-Z]+)\.adaptivecluster\.com/.exec(serviceUrl)
+    return envMatch ? envMatch['groups'].env : undefined
```

Comment:
Shouldn't use a static key as a dynamic key.
```suggestion
    return envMatch ? envMatch.groups.env : undefined
```

Reply:
this produces a ts error:

'groups' doesn't exist on RegExpExecArray

Reply:
https://github.com/microsoft/TypeScript/issues/29465#issuecomment-455489648